From 0b43954d80df3f2cb2661deb79fca2a8d8cb0ee2 Mon Sep 17 00:00:00 2001
From: John Youn <John.Youn@synopsys.com>
Date: Thu, 17 Dec 2015 11:18:13 -0800
Subject: [PATCH 153/194] usb: dwc2: Reduce delay when forcing mode in reset
Git-commit: 97e463886b873f62bea2293e7edf81fdb884b84f
Patch-mainline: v4.5-rc1
References: bsc#984114

The delay for force mode is only 25ms according to the databook.

Signed-off-by: John Youn <johnyoun@synopsys.com>
Signed-off-by: Felipe Balbi <balbi@ti.com>
Signed-off-by: Alexander Graf <agraf@suse.de>

---
 drivers/usb/dwc2/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/dwc2/core.c b/drivers/usb/dwc2/core.c
index 9dca835..39a0fa8 100644
--- a/drivers/usb/dwc2/core.c
+++ b/drivers/usb/dwc2/core.c
@@ -602,7 +602,7 @@ static void dwc2_clear_force_mode(struct dwc2_hsotg *hsotg)
 	 * NOTE: This long sleep is _very_ important, otherwise the core will
 	 * not stay in host mode after a connector ID change!
 	 */
-	usleep_range(150000, 160000);
+	msleep(25);
 }
 
 /*
-- 
1.8.5.6

